src/views/lighttable.c

```diff
@@ -1264,7 +1264,7 @@ int expose_full_preview(dt_view_t *self, cairo_t *cr, int32_t width, int32_t hei
     dt_mipmap_cache_get(darktable.mipmap_cache, NULL, preload_stack[count], mip, DT_MIPMAP_PREFETCH, 'r');
 
   lib->image_over = DT_VIEW_DESERT;
-  cairo_set_source_rgb(cr, .1, .1, .1);
+  cairo_set_source_rgb(cr, 0.2, 0.2, 0.2);
```

Comment:
I saw it, thanks. But I do want to see metadata I just don't want to see it over my image. I want to rate the image, add colors, etc and have the metadata right there while I do it without it blocking my view of the image I am rating. Having it hover in and out is a kludge.